lib/acme/client.rb

```diff
@@ -50,7 +50,8 @@ def initialize(jwk: nil, kid: nil, private_key: nil, directory: DEFAULT_DIRECTOR
 
   attr_reader :jwk, :nonces
 
-  def new_account(contact:, terms_of_service_agreed: nil)
+  def new_account(contact:, terms_of_service_agreed: nil, eab: nil, external_account_binding: nil)
```

Comment:
We could just force people to use `external_account_binding` instead of `eab`, I doubt it would change much.

Reply:
I agree